content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -408,3 +408,84 @@ second ensure that [keys and certificates are being delivered](#keys-and-certifi
 
 If everything appears to be working so far, the next step is to verify that the right [authentication policy](/docs/tasks/security/authentication/authn-policy/)
 is applied and the right destination rules are in place.
+
+### Auto mTLS
+
+If you suspect [auto mTLS](/docs/reference/glossary#auto-mtls) is not working
+as expected, please first read the
+[documentation](/docs/tasks/security/authentication/authn-policy/#auto-mutual-tls).
+
+Auto mTLS has a known limitation with [workload level](/docs/tasks/security/authentication/authn-policy/#enable-mutual-tls-per-workload) peer authentication.
+When workload level peer authentication is used, you need to configure a corresponding
+destination rule configuration to let client sidecar send mTLS or plaintext
+traffic correctly.
+
+If you still think there's an issue, follow the instructions below for
```

Comment:
```suggestion
If there still seems to be an issue, follow the instructions below to
```

Reply:
could we clarify if this is a temp limitation or permanent?  e.g. is this going to be resolved in a future release?

Reply:
removed. this is mentioned in the authn policy use reference here now.

putting aside, i don't think we have to make a statement on this limitation  in the troubleshooting guide anyhow.